tests/data/test2082

```diff
@@ -0,0 +1,55 @@
+<testcase>
+<info>
+<keywords>
+HTTP
+</keywords>
+</info>
+
+# Server-side
+<reply>
+<data nocheck="yes">
+HTTP/1.1 200 OK
+Date: Tue, 09 Nov 2010 14:49:00 GMT
+Server: test-server/fake
+Content-Type: text/html
+Content-Length: 0
+
+</data>
+</reply>
+
+# Client-side
+<client>
+<server>
+http
+</server>
+
+<name>
+Pre-request callback for HTTP
+</name>
+<tool>
+libprereq
+</tool>
+
+<command>
+%HOSTIP:%HTTPPORT/%TESTNUMBER
+</command>
+</client>
+
+# Verify data after the test has been "shot"
+<verify>
+<stripfile>
+s/^Connected to: .*/Connected to: testhost/
+s/^Connected from: .*/Connected from: testclient/
+s/^Remote port: \d+/Remote port: stripped/
+s/^Local port: \d+/Local port: stripped/
```

Comment:
The testcode and remote port shouldn't be needed to have stripped as we provide variables for them, don't we?

Reply:
I couldn't see any examples but maybe I wasn't looking hard enough...

Reply:
Found them. Adding them in now.